### DISCUSSION TITLE
Where do I find the files for this?

### DISCUSSION THREAD

**Discussion by pto199** [usememos/memos#4241] [Q&A]
I have this running on Unraid and I cannot find the files that make up the docker.

I see the database files in the containers /var/opt/memos (Which I have mapped to a different folder on my unraid machine) but I cant find the actual files.  They are supposed to be in the /.memos/ directory but I cant find this directory.  Its not on my unraid machine or inside the container

**Comment by RoccoSmit** [comment]
Which files are you looking for? Source code or uploaded files?

From the [docker file](https://github.com/usememos/memos/blob/main/Dockerfile) I believe the source code is compiled and stored in `/usr/local/memos` and your app data and uploaded files are stored in `/var/opt/memos`

Or are you trying to find your mapped directory? Docker desktop shows you where your files are mapped between the Docker system and your own under the `Bind mounts` tab 
![image](https://github.com/user-attachments/assets/bb3f3a8b-aaac-4800-a548-7f2bf968483a)
